### DISCUSSION TITLE
Jellyfin does not update

### DISCUSSION THREAD

**Discussion by albertusQuelle** [arabcoders/watchstate#568] [General] [answered]
Hi, 
i am using web-version. 
Plex as source and jellyfin as target. 

How can i fix that

`Episode Is not imported yet. Possibly because the backend was imported as metadata only.`

this was my command

`sudo docker exec -ti watchstate console state:export -vvif -s jellyfin`

**Comment by arabcoders** [comment, marked as answer]
hello,

Try this

1. add plex and import its data.
2. add jellyfin as metadata source only.
3. do `sudo docker exec -ti watchstate console state:export -vvif -s jellyfin`

some episodes probably will not sync as they use different GUIDs, to make sure visit one of the episodes in the history page and see if they both have metadata from both jellyfin and plex.
